src/hdf5_time_series_storage.jl

```diff
@@ -85,11 +85,22 @@ end
 
 get_file_path(storage::Hdf5TimeSeriesStorage) = storage.file_path
 
+function _time_array_wrapper_to_array(ta::TimeDataContainer)
+    # TODO: Implement for more dimensions.
+    # TODO: Is this storing the data efficiently?
+    if length(ta.data) == 1
+        return TimeSeries.values(first(values(ta.data)))
+    else
+        length(ta.data) > 1
```

Comment:
delete

Reply:
without this, a single time series get stored as a row and not as a column.

Reply:
This line is a no-op.  The expression returns a boolean but it is not captured.